src/custom-element.js

```diff
@@ -1291,3 +1242,25 @@ export function resetScheduledElementForTesting(win, elementName) {
 export function getElementClassForTesting(elementName) {
   return knownElements[elementName] || null;
 }
+
+/** @param {!Element} element */
+function assertNotTemplate(element) {
+  dev.assert(!element.isInTemplate_, 'Must never be called in template');
+};
+
+/**
+ * @return {!./service/vsync-impl.Vsync}
+ * param {!Element} element
+ */
+function getVsync(element) {
+  return vsyncFor(element.ownerDocument.defaultView);
+};
+
+/**
+ * Whether the element has been upgraded yet.
+ * @return {boolean}
+ * @final @this {!Element}
```

Comment:
just for my own knowledge, is it right that this function inherits the same signatures as the method?